scripts/convert_publish.py

```diff
@@ -82,12 +99,26 @@ def convert_posts(self, query = {}):
                 }
             
                 isClosedMessage = True
-                expiretion = timedelta(minutes = 30)
                 date_close = datetime.strptime("2106-02-07T06:28:15", '%Y-%m-%dT%H:%M:%S').isoformat()
                 if (doc["cashout_time"].isoformat() != date_close and doc["cashout_time"].isoformat() > datetime.now().isoformat()):
                     rshares_sum += cur_rshares_raw
                     pool["state"]["msgs"] += 1
                     isClosedMessage = False
+                    
+                    delay_trx = {
+                      "trx_id": "",
+                      "sender": "gls.publish",
+                      "sender_id": hex(utils.convert_hash(doc["permlink"]) << 64 | utils.string_to_name(doc["author"])),
+                      "payer": "gls.publish",
+                      "delay_until" : doc["cashout_time"].isoformat(), 
+                      "expiration" :  doc["created"].isoformat() + expiretion, 
```

Comment:
Why do you make `isoformat` and then concatenate with `expiretion`?
And may be in this case need to use `cashout_time`?